CHANGES.md

```diff
@@ -75,6 +75,21 @@ OpenSSL 3.2
 
 ### Changes between 3.2.0 and 3.2.1 [xx XXX xxxx]
 
+ * The function EVP_PKEY_public_check() performs various checks on the RSA
+   public key. Among these checks, a computation is done to confirm that the
+   RSA modulus, n, is composite.  For valid RSA keys, n is a product of two
+   or more large primes and this computation completes quickly. However, if
+   n is an overly large prime, then this computation would take a long time.
+
+   An application that calls EVP_PKEY_public_check() and supplies an RSA key
+   obtained from an untrusted source could be vulnerable to a Denial of Service
+   attack.
```

Comment:
This needs at least information about internal use of that function in OpenSSL.

Reply:
Added